Flat/src/ModulesLepPho.cc

```diff
@@ -269,35 +391,6 @@ void PandaAnalyzer::SaveGenLeptons()
 
 }
 
-void PandaAnalyzer::LeptonSFs()
```

Comment:
I want this function back.

Reply:
Okay, I will bring it back and book/fill the sf_lepID/sf_lepIso branches if not using complicated leptons.